src/Waargonaut/Encode.hs

```diff
@@ -204,6 +207,10 @@ integral = encToJsonNoSpaces _JNum (review _JNumberScientific . fromIntegral)
 bool :: Applicative f => Encoder f Bool
 bool = encToJsonNoSpaces _JBool id
 
+-- | Encode a 'String'
+string :: Applicative f => Encoder f String
```

Comment:
Can you add the `string'` version as well, that specialises the `f` to `Identity`. Similar to `text'` and friends.